message/messaging_service.hh

```diff
@@ -341,9 +341,9 @@ public:
     future<> send_repair_put_row_diff(msg_addr id, uint32_t repair_meta_id, repair_rows_on_wire row_diff);
 
     // Wrapper for REPAIR_ROW_LEVEL_START
-    void register_repair_row_level_start(std::function<future<> (const rpc::client_info& cinfo, uint32_t repair_meta_id, sstring keyspace_name, sstring cf_name, dht::token_range range, row_level_diff_detect_algorithm algo, uint64_t max_row_buf_size, uint64_t seed, unsigned remote_shard, unsigned remote_shard_count, unsigned remote_ignore_msb, sstring remote_partitioner_name, table_schema_version schema_version)>&& func);
+    void register_repair_row_level_start(std::function<future<> (const rpc::client_info& cinfo, uint32_t repair_meta_id, sstring keyspace_name, sstring cf_name, dht::token_range range, row_level_diff_detect_algorithm algo, uint64_t max_row_buf_size, uint64_t seed, unsigned remote_shard, unsigned remote_shard_count, unsigned remote_ignore_msb, table_schema_version schema_version)>&& func);
```

Comment:
This change will cause compatibility issue. We can only add new parameters ad rpc::optional and we can not remove old parameters. If  remote_partitioner_name is not needed any more, leave as it is in the rpc verb.

Reply:
Thanks @asias - I was expecting difficulties around this place :). I will do as you suggested. Leave the parameter but just ignore it.

Reply:
Yes. After this change, we still have the partitioner name, right? If this is the case, we need to send the partitioner name. We need to consider the case when the remote is a old node does not support this change. Will mixed node cluster work?

Reply:
That's a fair point. I will always send "Murmur3" from "new" nodes to be compatible with old nodes in mixed cluster. That should do the trick.